core/src/main/scala/filodb.core/store/ChunkSetInfo.scala

```diff
@@ -348,6 +350,15 @@ class CountingChunkInfoIterator(base: ChunkInfoIterator,
     // Why two counters ?
     // 1. query stats counter to meter usage by ws/ns. This will eventually be sent as part of query result.
     dataBytesScannedCtr.addAndGet(bytesRead)
+    if (dataBytesScannedCtr.get() > maxBytesScanned) {
+      val exMessage = s"Actual raw data bytes scan of ${dataBytesScannedCtr.get()} bytes exceeds limit of " +
+        s"${maxBytesScanned} bytes queried per shard. " +
+        s"Try one or more of these: " +
+        s"(a) narrow your query filters to reduce to fewer time series matches " +
```

Comment:
NIT: Is it narrow timeseries matches by adding filters?

Reply:
will add the proper comment in the next iteration, otherwise the approval will disappear